src/main/webapp/scripts/user-script.js

```diff
@@ -1,7 +1,14 @@
 var params = JSON.parse(localStorage.getItem('oauth2-test-params'));
 var token = params['access_token'];
+var domain = localStorage.getItem('domain');
 var flatdata = []
 var data = {}
+var input;
+
+function userOnload(){
```

Comment:
is this function needed if its just calling two other functions and only called once, can those other two functions just be called where userOnload() is called?